### PR TITLE
chore: prepare for the 0.8.x series

### PR DESCRIPTION
**Pull request by ggreif**

### CODE REVIEW

File: Building.md [hunk 1]
```diff
@@ -54,11 +55,11 @@ For more details on our CI and CI setup, see `CI.md`.
 
 ## Making releases
 
-We make frequent releases, at least weekly. The steps to make a release (say, version 0.7.1) are:
+We make frequent releases, at least weekly. The steps to make a release (say, version 0.8.1) are:
```
Comment (crusso) [inline Building.md:58]:
```suggestion
We make frequent releases, at least weekly. The steps to make a release (say, version 0.8.0) are:
```

File: Building.md [hunk 2]
```diff
@@ -54,11 +55,11 @@ For more details on our CI and CI setup, see `CI.md`.
...
 
  * Make sure that the top section of `Changelog.md` has a title like
 
-        ## 0.7.1 (2022-08-25)
+        ## 0.8.1 (2023-01-25)
```
Comment (crusso) [inline Building.md:62]:
```suggestion
        ## 0.8.0 (2023-01-25)
```

File: Building.md [hunk 3]
```diff
@@ -74,16 +75,16 @@ We make frequent releases, at least weekly. The steps to make a release (say, ve
 
  * Define a shell variable `export MOC_MINOR=1`
```
Comment (crusso) [inline Building.md:76]:
```suggestion
 * Define a shell variable `export MOC_MINOR=0`
```

File: Building.md [hunk 4]
```diff
@@ -74,16 +75,16 @@ We make frequent releases, at least weekly. The steps to make a release (say, ve
...
 
- * Look at `git log --first-parent 0.7.$(expr $MOC_MINOR - 1)..HEAD` and check
+ * Look at `git log --first-parent 0.8.$(expr $MOC_MINOR - 1)..HEAD` and check
```
Comment (crusso) [inline Building.md:78]:
```suggestion
 * Look at `git log --first-parent 0.8.$(expr $MOC_MINOR - 1)..HEAD` and check
```
I guess this won't work if we start from 0.

**Review by crusso — APPROVED** [review]
I suggested starting the last part of the version form 0 (not 1) but maybe that messes up your scripts.

Feel free to ignore all the suggestions.

**Comment by ggreif** [comment]
> I suggested starting the last part of the version form 0 (not 1) but maybe that messes up your scripts.

Yeah, the `1` in the file is only an example. I'll start with 0.8.0 the doing the release, and take care of the problematic `expr`.

**Pull request merged**